src/arrayMethodSort.js

```diff
@@ -5,7 +5,30 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const sortedArray = this;
+
+    let waschange = false;
+
+    do {
+      waschange = false;
+
+      for (let i = 1; i < sortedArray.length; i++) {
+        if (compareFunction === undefined) {
+          if (sortedArray[i - 1].toString() > sortedArray[i].toString()) {
+            waschange = true;
+
+            sortedArray.splice(i - 1, 2, sortedArray[i], sortedArray[i - 1]);
```

Comment:
Not consistent spacing
```suggestion
            waschange = true;
            sortedArray.splice(i - 1, 2, sortedArray[i], sortedArray[i - 1]);
```